api/envoy/type/tracing/v3/custom_tag.proto

```diff
@@ -96,7 +96,10 @@ message CustomTag {
     // A request header custom tag.
     Header request_header = 4;
 
+    // A response header custom tag.
+    Header response_header = 5;
+
     // A custom tag to obtain tag value from the metadata.
-    Metadata metadata = 5;
+    Metadata metadata = 6;
```

Comment:
Please don't update API in this way, we should never change ID of a exist field. And if there are no related implementation, please use `[#not-implemented-hide:]` to hide this API.

Reply:
thanks , fix for not change proto id.   and  i will implemented later .  so should i need to add not-implemented-hide tag?

Reply:
if you will add the implementation in this PR, then you needn't to do that.